mc/Mongo-Tests-Core.package/MongoReplicationOkTest.class/instance/testReplicaSetStepDown.st

```diff
@@ -1,19 +1,23 @@
 tests
 testReplicaSetStepDown
 
-	"See _manual_set_up_ to set up the testing mongodb replica set."
-	isMongoAvailable ifFalse: [ ^self skip ].
-
+	"Step down the (prioritary) primary during some seconds"
 	mongo replicaSetStepDown: 10 seconds.
 
+	"It's still connected but it's not primary."
 	self assert: mongo isOpen.
 	
+	"Then, it will signal an error on a write operation."
 	[ 	self addNewCollection.
 		self fail: 'should raise a `not master` error' ]
 			on: MongoCommandError
 			do: [ :error | 
 				self assert: error isNotMaster ].
-			
-	20 seconds wait.
+
+	"When the step down seconds passed, and the election finished, 
+	the former primary server will be the primary again because it has more priority
+	(see the replication scenario)."
+	30 seconds wait.
```

Comment:
I have seen a re-election take longer than this as well. Can we somehow actively wait for it instead of making this system test always take X seconds?

Reply:
In one hand, we can tune the testing mongodb setup with sorter times, in other hand the test can have a loop that checks often if the wait can end with a timeout (30 seconds in this case). I will create an issue with this.

Reply:
Issue #55